### ISSUE TITLE
Add .otc export support

### ISSUE THREAD

**Issue by MaddTheSane** [fontforge/fontforge#3543]
As it stands,  FontForge does not support generation of OpenType Collection font format (OpenType PostScript collections). This was introduced in OpenType 1.8 and is an expansion of the TrueType Collection font format. Good news is, FontForge can open otc fonts right now, it just can't save them.

### UI Changes needed:
- [ ] Rename "Generate TTC..." to something more appropriate. ("Generate Collection...", maybe?)
- [ ] Add "OpenType (OTC)" to the font format selection menu.
- [ ] \(minor) recognize otc files as a font file in the open dialog, including icon.

**Comment by khaledhosny** [comment]
FontForge can generate TTC fonts with CFF tables just fine (you need to check “As CFF fonts” in the Generate TTC dialog, so what is missing is mostly GUI touch-ups to make this more obvious.

**Comment by MaddTheSane** [comment]
I tested it: enabling it didn’t enable the PostScript options nor disable the “Generate TrueType Hints” checkmark.

**Comment by khaledhosny** [comment]
But it did generate a font collection with CFF tables, right?

**Comment by MaddTheSane** [comment]
Yes, it generates OpenType PostScript collections. So just some UI improvements would be needed.
